File: GullSharksLib/Models/FriendsList.cs
namespace GullSharksLib;

public class FriendsList
{
    public int ID { set; get; }
    public int User_ID { set; get; }
    public int Friend_ID { set; get; }
    public bool IsConfirmed { get; set; }
    public bool IsDeleted { get; set; }
    public DateTime DateAdded { set; get; }
}

File: GullSharksWeb/src/models/FriendsList.ts
import { User } from "./User";
import { UserDetails } from "./UserDetails";

export interface FriendsList {
  id: number;
  user_ID: number;
  friend_ID: number;
  isConfirmed: boolean;
  isDeleted: boolean;
  dateAdded: Date;
  user?: User;
  userDetails?: UserDetails;
}
